andela/travel_tool_front
# #160096049 Fix open requests count

#### What does this PR do?
Fixes the bug in the open requests count so that it displays the accurate number.

#### Description of Task to be completed?
- Modify action creators for fetching requests to use new response from the API
- Update reducers with the new response from the actions
- Replace the url from the server with url compiled from the client-side
- Fix tests and add more tests for added code
- Update functions for the table filters to use query built from client

#### How should this be manually tested?
- Clone the repo -` git clone https://github.com/andela/travel_tool_front.git`
- Enter into the project directory - `cd travel_tool_front`
- Start the app using `make start`
- Start the API - see (https://github.com/andela/travel_tool_back/pull/24)
- Open the browser and navigate to the Requests URL `http://127.0.0.1:3000/requests`
- You should see the requests count displaying correctly.

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[#160096049](https://www.pivotaltracker.com/story/show/160096049)

#### Screenshots (if appropriate)
None

#### Questions: 
N/A
